nautobot_ssot/forms.py

```diff
@@ -24,7 +24,7 @@ class SyncLogEntryFilterForm(BootstrapMixin, forms.ModelForm):
     """Form for filtering SyncLogEntry records."""
 
     q = forms.CharField(required=False, label="Search")
-    sync = forms.ModelChoiceField(queryset=Sync.objects.all(), required=False)
+    sync = forms.ModelChoiceField(queryset=Sync.objects.defer("diff").all(), required=False)
```

Comment:
Do we need to `defer` or can we outright use `only` instead? From the docs
> only use defer() when you cannot, at queryset load time, determine if you will need the extra fields or not
This would mean that we _might_ need the diff for the filter form. Is this the case?

Reply:
May not work, because `values` returns a dictionary, but I think we should evaluate this.

Reply:
specifically, when trying `.values()` I get the error:
```
'dict' object has no attribute 'get_source_url'
```